pkg/install/install.go

```diff
@@ -431,8 +436,8 @@ func (o *Install) runWizard() error {
 func (o *Install) runEverestWizard() error {
 	var namespaces string
 	pNamespace := &survey.Input{
-		Message: "Namespaces managed by Everest (comma separated)",
-		Default: namespaces,
+		Message: "Namespaces managed by Everest [comma separated]",
+		Default: DefaultEverestNamespace,
```

Comment:
This will make the default value respect the `--namespaces` flag, if provided.

```suggestion
		Default: o.Config.Namespaces,
```